homeassistant/components/github/sensor.py

```diff
@@ -136,6 +139,10 @@ def update(self):
         self._available = self._github_data.available
         self._latest_commit_message = self._github_data.latest_commit_message
         self._latest_commit_sha = self._github_data.latest_commit_sha
+        if self._latest_release_url is not None:
+            self._self_latest_release_tag = self._latest_release_url.split("tag/")[1]
```

Comment:
Typo. Change to `self._latest_release_tag`.

Reply:
check, will do